web.config

```diff
@@ -7,7 +7,7 @@
     <httpPlatform stdoutLogEnabled="true"
                   stdoutLogFile="%HOME%\LogFiles\application\stdout"
                   processPath="%JAVA_HOME%\bin\java.exe"
-        arguments="-Djava.net.preferIPv4Stack=true -Dserver.port=%HTTP_PLATFORM_PORT% -jar &quot;%HOME%\site\wwwroot\claim-store.jar&quot;">
+        arguments="-javaagent:&quot;%HOME%\site\wwwroot\applicationinsights-agent-2.0.2.jar&quot; -Djava.net.preferIPv4Stack=true -Dserver.port=%HTTP_PLATFORM_PORT% -jar &quot;%HOME%\site\wwwroot\claim-store.jar&quot;">
```

Comment:
Ah that is what's missing from agent additional config 👏 . Can we have an update in `java-logging-appinsights` readme?

By the way, agent jar comes together from library, is it not visible / accessible just from there? Then could get rid of extra jar in project root lib folder

Reply:
Not without unzipping it and extracting the jar out, I'll remove it from java-logging after I've got this to prod.
Will be updating rhubarb and the onboarding docs as well as java-logging